docs/CONTRIBUTING.md

```diff
@@ -53,6 +53,16 @@ These types of issue cover everything that involves adding new features to SORMA
 * **Possible Alternatives:** If possible, provide alternative solutions in case your proposed feature can not be implemented in the way you outlined it for any reason. This is generally *optional* but might be requested at a later point in time.
 * **Additional Information:** If there is anything else you want to add to your request, you can put it here. *Optional*.
 
+### Task
+
+Something to be done but not directly changes anything the product. Examples: Preparing coming up features/changes, change processes on Github etc.
```

Comment:
Proposed text:

A task that needs to be done but does not directly change anything about the product. This could for example be the preparation of an upcoming feature/change, optimization of processes on GitHub, or update of one of the guides or Wiki articles in this repository.